src/components/graph-view.js

```diff
@@ -850,6 +864,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     this.setState(newState);
 
     if (!creatingEdge) {
+      this.asyncElevateNodeAndEdges(node);
```

Comment:
I am wondering how will this  work for multiple selection of node

Reply:
Users must select one node at a time so they'll be moved up into the view one by one. In general, the last selected (clicked on) will be at the top.